tools/cli/.goreleaser.yml

```diff
@@ -0,0 +1,64 @@
+# Visit https://goreleaser.com for documentation on how to customize this
+# behavior.
+project_name: openapicli
```

Comment:
we should come up with a better name for the cli

Reply:
mongoapi?

Reply:
I opened a pool on the team slack channel, I will create a ticket to rename the cli after everyone has voted